spec/services/api/v3/ecf/participants_query_spec.rb

```diff
@@ -38,12 +38,47 @@
       end
     end
 
-    context "with cohort filter" do
+    context "when filtering by cohort" do
       let(:params) { { filter: { cohort: cohort.start_year.to_s } } }
 
-      it "returns all users that belong to a provider" do
+      it "only returns users with induction records in the specified cohort" do
         expect(subject.participants_for_pagination).to match_array([user])
       end
+
+      context "when the user has induction records against multiple cohorts" do
+        let(:new_cohort) { Cohort.next }
+
+        before do
+          create(:school_cohort,
+                 default_induction_programme: induction_programme,
```

Comment:
I think we need to be careful here as we're linking this school cohort to the existing induction programme, which is linked to another partnership in another cohort. Is there a way we can restructure this setup so all the records are linked to the same cohort?